mleap-xgboost-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/mleap/XGBoostClassificationModelParitySpec.scala

```diff
@@ -46,30 +48,51 @@ class XGBoostClassificationModelParitySpec extends FunSpec
     "nworkers" -> 2
   )
 
-  val dataset: DataFrame = {
+  val denseDataset: DataFrame = {
     SparkParityBase.dataset(spark).select("fico_score_group_fnl", "dti").
       filter(col("fico_score_group_fnl") === "500 - 550" ||
         col("fico_score_group_fnl") === "600 - 650")
   }
 
-  val sparkTransformer: Transformer = {
+  val mixedDataset: DataFrame = {
```

Comment:
This is supposed to be a Sparse dataset at some point along the way right ? Due to `implicit class VectorOps` being implicit I am not sure where that happens

Reply:
I called it a mixed Dataset because it has both sparse and dense rows. I can rename it sparseDataset to be more clear

Reply:
Mmh, I don't know if a dataset having both sparse rows and "full" rows would still sparse or dense. Is the glass half empty or half full?
But I think for readability it's good to have a sparkeDataset and a denseDataset, so the tests are clearer

Reply:
updated naming to sparseDataset